miner/worker.go

```diff
@@ -1466,8 +1483,22 @@ func (w *worker) commitNewWork(interrupt *int32, noempty bool, timestamp int64)
 	l2CommitNewWorkTidyPendingTxTimer.UpdateSince(tidyPendingStart)
 
 	var skipCommit, circuitCapacityReached bool
+
+	commitSystemTxStart := time.Now()
+	if w.chainConfig.Scroll.SystemTxEnabled() {
+		log.Trace("Processing SystemTxs for inclusion", "count", len(systemTxs))
+
+		txs := types.NewOrderedSystemTxs(systemTxs)
+		skipCommit, circuitCapacityReached = w.commitTransactions(txs, w.coinbase, interrupt)
```

Comment:
We need to carefully review how `commitTransactions` behaves with system txs. But this depends on further design, so we can do it in another PR.
1. Should we exclude them from the `IsValidBlockSize` check? (Probably yes, we won't include the full tx in DA)
2. Should we use the same "skip" behavior (for CCC and other errors like `ErrGasLimitReached`) as we do for `L1MessageTx`? The `skipBitmap` in DA currently only works for L1 message txs.

Reply:
For DA just the system contract and call data should be sufficient, no? We don't have anything else.

Reply:
1. We should not.
2. We can verify that the `SystemTx` always won't be skipped since we generate these txs.